spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -71,4 +71,40 @@
       end
     end
   end
+
+  describe 'GET #index/search' do
+    context 'given VALID params' do
+      it 'returns a 200 status code' do
+        url = 'https://google.com'
+        keyword = Fabricate(:parsed_keyword) { keyword_search_entries(count: 0) }
+        Fabricate.times(2, :keyword_search_entry, urls: [url], keyword_id: keyword.id)
+        Fabricate(:keyword_search_entry, urls: ['https://ruby-lang.org'], keyword_id: keyword.id)
+
+        get(search_api_v1_keywords_path, params: { search: 'google', query_type: 'partial' })
+
+        expect(response).to(have_http_status(:success))
+      end
+
+      it 'returns the search result' do
```

Comment:
Just minor suggestion to name the example:

```suggestion
      it 'returns a list of search results' do
```